packages/cli/src/standalone-impl.ts

```diff
@@ -13,32 +15,90 @@ import {
 } from '@pipcook/pipcook-core';
 import * as fs from 'fs-extra';
 import * as path from 'path';
-export class DataAccessorImpl<T> implements DataAccessor<T> {
+export class DataAccessorImpl<T> implements RandomDataAccessor<T> {
+  dataLength: number;
+  randomIndex: number;
+  randomMap: Array<number>;
   constructor(
     private dataAccessor: DataAccessor<T>
   ) {}
+  init(size: number): void {
+    this.dataLength = size;
+    this.randomIndex = 0;
+    this.randomMap = this.createRandom(this.dataLength);
+  }
   next(): Promise<Sample<T> | null> {
     return this.dataAccessor.next();
   }
   nextBatch(batchSize: number): Promise<Array<Sample<T>> | null> {
     return this.dataAccessor.nextBatch(batchSize);
   }
+  nextRandom(): Promise<Sample<T> | null> {
+    if ( this.randomIndex >= this.dataLength ) {
+      this.dataAccessor.seek(this.randomIndex++);
```

Comment:
```suggestion
      return null;
```

Reply:
I want to simulate the behavior like Function next().